assets/js/mbta.ts

```diff
@@ -2761,13 +2806,27 @@ const stationConfig: {
           },
         },
       },
+      {
+        id: 'SLEC',
+        name: 'Lechmere - GLX',
+        zones: {
+          m: {
+            modes: {
+              auto: false,
+              custom: true,
+              headway: false,
+              off: true,
+            },
+          },
+        },
+      },
       {
         id: 'GLEC',
         name: 'Lechmere',
         zonePositions: {
           left: ['n'],
           center: [],
-          right: ['m'],
+          right: ['s'],
```

Comment:
Updated this one because there was an Id colliding with the new GLEC-m zone on the new train stop, and to more accurately map s to south.